test/components/CheckboxGroup.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { describe, it, expect, test } from 'vitest'
+import RadioGroup, { type RadioGroupProps, type RadioGroupSlots } from '../../src/runtime/components/RadioGroup.vue'
```

Comment:
```suggestion
import CheckboxGroup, { type CheckboxGroupProps, type CheckboxGroupSlots } from '../../src/runtime/components/CheckboxGroup.vue'
```

The rest of the test should also be updated - it's currently testing the wrong component.